### PR TITLE
feat: (L1) Add log level flag

### LINKED ISSUE #692: Add a log level flag
When running the client without a log level, the default should be `info`

We should add a flag on the cli to set a custom log level. This should be a global flag.

We should tests this against Kurtosis and Hive. We need changes to the dockerfiles to ensure that we are sending the proper flag when initialing the clients.

### PR DESCRIPTION
**Pull request by belfortep**
**Motivation**

Add log level flag to the `CLI`

**Description**

When running the program, now you can pass the `--log-level` flag, if it isn't used, the log-level will be `INFO` as default

Closes #692

### CODE REVIEW

File: cmd/ethereum_rust/cli.rs [hunk 1]
```diff
@@ -19,6 +19,7 @@ pub fn cli() -> Command {
                 .value_name("PORT")
                 .action(ArgAction::Set),
         )
+        .arg(arg!(-v --verbose "Max verbosity for logs").required(false))
```
Comment (mpaulucci) [inline cmd/ethereum_rust/cli.rs:22]: Sorry, I was thinking something more like what lighthouse has

```
      --logfile-debug-level <LEVEL>
          The verbosity level used when emitting logs to the log file. [default:
          debug] [possible values: info, debug, trace, warn, error, crit]
```

the flag name could be `--log-level`. Thoughts?

Comment (belfortep) [reply cmd/ethereum_rust/cli.rs:22]: Oh okay, changing that now to match more levels of logs 🫡

File: cmd/ethereum_rust/ethereum_rust.rs [hunk 2]
```diff
@@ -45,6 +40,13 @@ async fn main() {
         return;
     }
 
+    let log_level = matches
+        .get_one::<String>("log-level")
+        .expect("shouldn't happen, log-level is used with a default value");
+    let log_level = Level::from_str(log_level).unwrap_or(Level::INFO);
```
Comment (fkrause98) [inline cmd/ethereum_rust/ethereum_rust.rs:46]: maybe we should fail if the given value for the log-level is not correct instead?
this is parsed when the node starts, right?

Comment (belfortep) [reply cmd/ethereum_rust/ethereum_rust.rs:46]: Yes this is when the node starts, thought that as logs aren't really needed for the execution, we could have a default value.
Changing it to an `expect` with some error description

Comment (fkrause98) [reply cmd/ethereum_rust/ethereum_rust.rs:46]: Ok, I agree with that approach.

**Review by fkrause98 — COMMENTED** [review]
Looks good, let's update the readme and we're done here.

**Review by fkrause98 — APPROVED** [review]

**Pull request merged**